src/ebay-page-notice/page-notice.tsx

```diff
@@ -1,44 +1,70 @@
-import React, { FC, ReactElement } from 'react'
+import React, { Dispatch, FC, ReactElement, useState } from 'react'
 import NoticeContent from '../common/notice-utils/notice-content'
 import { EbayNoticeContent } from '../ebay-notice-base/components/ebay-notice-content'
 import { EbayIcon, Icon } from '../ebay-icon'
 
 type Props = React.HTMLProps<HTMLElement> & {
     status?: 'general' | 'attention' | 'confirmation' | 'information',
-    'aria-label'?: string;
+    'aria-label'?: string,
+    a11yDismissText: string,
+    onDismissed?: Dispatch<boolean>
```

Comment:
I think we should use the same name as marko version for consistency: `onDismiss`
Also can we pass the event instead of the boolean to follow marko API?

Reply:
Updated to onDismiss and now dispatches a MouseEvent.